src/shoppingList/controllers/FindShoppingListByIdController.js

```diff
@@ -0,0 +1,21 @@
+import { findShoppingListById } from "../repository/shoppingListRepository.js";
+import { StatusCodes } from "http-status-codes";
+
+export class FindShoppingListByIdController {
```

Comment:
Você não devia criar uma classe nova para todos os requests que você for fazer. Pode parecer que você está separando/organizando o código, mas você só está criando mais boilerplate. Você pode separar tudo criando routers e passando os routers para o `app`.

```js
const express = require('express')
const router = express.Router()

// middleware that is specific to this router
router.use((req, res, next) => {
  console.log('Time: ', Date.now())
  next()
})
// define the home page route
router.get('/', (req, res) => {
  res.send('Birds home page')
})
// define the about route
router.get('/about', (req, res) => {
  res.send('About birds')
})

module.exports = router
```

E em `main.js`:

```js
const birds = require('./birds')

// ...

app.use('/birds', birds)
```

[Mais informações sobre routing](https://expressjs.com/en/guide/routing.html).

Reply:
Hey, obrigado pelo feedback. 
A ideia seria dps refatorar este projeto, adotando as melhores rpaticas.
Acredito que por conta do prazo que tenho e afins fique inviavel eu implementar essa melhoria

Reply:
Só para entender melhor, a ideia seria ter um arquivo de rotas para cada contexto ? 
Tipo um arquivo de rotas para User, ShoppingList, ShoppingListProducts e dps importa dodos eles no arquivo princiapl ?